client/src/pages/HomePage/HomePage.js

```diff
@@ -1,5 +1,107 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable react/jsx-props-no-spreading */
 import React from 'react';
+import { Button, Container, Typography } from '@mui/material';
+import ArrowRightAltIcon from '@mui/icons-material/ArrowRightAlt';
+import { styled } from '@mui/material/styles';
+import { ImageCarousel } from '../../components';
+import styles from './HomePage.module.scss';
 
-const HomePage = () => <div>HOME</div>;
+const TravelText = styled((props) => <Typography variant="h1" {...props} />)(({ theme }) => ({
+  fontWeight: 700,
+  fontSize: '36px',
+  lineHeight: '44px',
+  color: 'white',
+
+  position: 'relative',
+  margin: 0,
+  marginBottom: '12px',
+
+  [theme.breakpoints.up('tablet')]: {
+    fontSize: '48px',
+    lineHeight: '59px',
+  },
+
+  [theme.breakpoints.up('laptop')]: {
+    fontSize: '72px',
+    lineHeight: '88px',
+  },
+
+  '.heroHeaderHighlighted': {
+    fontFamily: 'Pacifico',
+    fontWeight: 400,
```

Comment:
Suggest deleting other unused font weights from this font

Reply:
Will do on future fixes when we are totally done with our design.